### ISSUE TITLE
[DecorativeIcon] add Magnifying glass

### ISSUE THREAD

**Issue by elliottjro** [telus/tds-core#1191]
## Description

Currently, there is an icon available as a decorative icon
`<DecorativeIcon symbol="spyglass" />`

but none available for SVG Icon

## Recommendation

Create a spyglass icon in SVG Icon format

## Designs

Proposed SVG:
```
<?xml version="1.0" encoding="UTF-8"?>
<svg width="24px" height="24px" viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
    <!-- Generator: Sketch 55.2 (78181) - https://sketchapp.com -->
    <title>Magnifier</title>
    <desc>Created with Sketch.</desc>
    <g id="Welcome" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
        <g id="Artboard">
            <g id="Magnifier">
                <path d="M23.0406151,23.0406151 L15.5,15.5" id="Line" stroke="#4B286D" stroke-width="1.2" stroke-linecap="round"></path>
                <path d="M9,18 C13.9705627,18 18,13.9705627 18,9 C18,4.02943725 13.9705627,0 9,0 C4.02943725,0 0,4.02943725 0,9 C0,13.9705627 4.02943725,18 9,18 Z M9,16.8 C4.69217895,16.8 1.2,13.307821 1.2,9 C1.2,4.69217895 4.69217895,1.2 9,1.2 C13.307821,1.2 16.8,4.69217895 16.8,9 C16.8,13.307821 13.307821,16.8 9,16.8 Z" id="Oval" fill="#4B286D" fill-rule="nonzero"></path>
            </g>
        </g>
    </g>
</svg>
```

## Meta

- Willing to develop solution: Yes
- Has workaround: Yes -> save icon standalone in project
- High impact: Yes - spyglass icon is the only one missing
- Designer/Developer Pair: Elliott / Ally Noormohamed

**Comment by varunj90** [comment]
This is part of TDS backlog and will be done in 1st half of August: https://telusdigital.atlassian.net/browse/TDS-1406

**Comment by varunj90** [comment]
we will confirm this with @gkychow and get back to you.

cc: @Christina-Lo

**Comment by Christina-Lo** [comment]
@ElliottJRo where is this specifically going to be used? Spyglass is being added into our SVG icons, however it is reserved for search/search bar only. It will not be available as a "decorative svg icon" 

> Decorative icons do not perform a role beyond visual aesthetics and are hidden from screen readers using the aria-hidden attribute. Usually, the information being communicated with the icon is also conveyed in another manner.

https://tds.telus.com/components/index.html#decorativeicon

**Comment by ElliottJRo** [comment]
cc: @allynoormohamed 

I think Ally would be able to give more info regarding this.

**Comment by Christina-Lo** [comment]
@ElliottJRo 
The decorative icons are not meant to be Standalone as they do not meet some accessibility concerns, we are reviewing Standalone icons with several teams, UPC first as they will be owning the global widgets (search, cart, support). Will also look into these and keep you updated.

**Comment by theetrain** [comment]
Update: the TDS team will implement this.

**Comment by theetrain** [comment]
This is now available in [@tds/core-interactive-icon@1.3.0](https://github.com/telus/tds-core/releases/tag/%40tds%2Fcore-interactive-icon%401.3.0)
Documentation: https://tds.telus.com/components/index.html#navbutton

Magnifying Glass along with the other 'NavButton' interactive icons can be used in a navigational header.
